### ISSUE TITLE
Write big file when openAsync 

### ISSUE THREAD

**Issue by fftidus** [Gamua/Adobe-Runtime-Support#134]
## Problem Description

var file:File=File.applicationStorageDirectory.resolvePath("testA.swf");
var fileStr:FileStream = new FileStream();
fileStr.openAsync(file,FileMode.WRITE);
fileStr.addEventListener(OutputProgressEvent.OUTPUT_PROGRESS,onWriteEnd);
fileStr.addEventListener(IOErrorEvent.IO_ERROR,onWriteError);
fileStr.writeBytes(bytes, 0,bytes.length);
function onWriteEnd(e):void{
	trace("write end");
}
function onWriteError(e):void{
	trace("write error");
}

When the bytes need to write is big (test with 500k), there is no result and error.

Describe your problem in detail. Include the following information:

* Which AIR SDK did you use? Hint: always try the latest version from [Adobe Labs]
AIR33  Android9

**Comment by ajwfrost** [comment]
Yes we're looking into this, looks like another async issue with the background thread not getting released to complete the write. Interesting that you don't get any outputProgress event though, we'd seen one of these be sent and then nothing else..

thanks

**Comment by Ender22** [comment]
Not sure if related, but on air 33 when I am downloading files around 1.2mb the app appears to download all of the data (the urlstream bytesAvailable matches the bytesTotal) but then when writing the file, the FileStream Close event never fires and only about 300kb of data gets written.

**Comment by ajwfrost** [comment]
Yes that would be the same problem - solved now, we'll release the fix soon..
thanks

**Comment by itlancer** [comment]
May be related issue: https://github.com/Gamua/Adobe-Runtime-Support/issues/97

**Comment by AditivoInteractiveGroup** [comment]
Hi @ajwfrost ! :) We are experimenting the same issue with openasync ando the 33 sdk. Pracrically allá the project relies on apenasync, since we are downloading lots of files from a webserver. Any idea when Will the next release will take place ? We are really near of the release date for this app. And se just need to know ir de wait for this fix or we move in another direction.  Thanks !!

**Comment by ajwfrost** [comment]
We’d been holding off as the Android App Bundle work looked like it was going to be complete last week  .. but we faced some issues with resources from ANEs in that, so are going ahead with the next release to fix this and a couple of other issues. Targeting Wednesday..

**Comment by itlancer** [comment]
@fftidus, for me related issue (https://github.com/Gamua/Adobe-Runtime-Support/issues/97) works fine with latest AIR 33.1.0.50 for Android and iOS.
